server/src/main/java/org/elasticsearch/index/mapper/MappedFieldType.java

```diff
@@ -119,15 +115,13 @@ public boolean equals(Object o) {
             docValues == fieldType.docValues &&
             Objects.equals(name, fieldType.name) &&
             Objects.equals(indexAnalyzer, fieldType.indexAnalyzer) &&
-            Objects.equals(searchAnalyzer, fieldType.searchAnalyzer) &&
-            Objects.equals(searchQuoteAnalyzer(), fieldType.searchQuoteAnalyzer()) &&
             Objects.equals(eagerGlobalOrdinals, fieldType.eagerGlobalOrdinals) &&
             Objects.equals(meta, fieldType.meta);
```

Comment:
Should `TestSearchInfo` be in `equals` and `hashCode`?

Reply:
`equals` and `hashCode` on `MappedFieldType` are basically useless now.  Previously we used them to check that fields with the same name defined on different types within a single index were identical, but we only have one type per index and this check doesn't exist any more since #58162 .  I'll open a separate issue to remove them.